Without this patch bmake does not compile succesfully when
USE_FILEMON is not set. This has been reported upstream via mail.

diff -upr bmake.orig/meta.c bmake/meta.c
--- bmake.orig/meta.c	2020-02-12 13:11:41.903373030 +0100
+++ bmake/meta.c	2020-02-12 13:12:20.140056931 +0100
@@ -1690,7 +1690,11 @@ meta_compat_parent(pid_t child)
     meta_job_parent(NULL, child);
     close(childPipe[1]);			/* child side */
     outfd = childPipe[0];
+#ifdef USE_FILEMON
     metafd = Mybm.filemon ? filemon_readfd(Mybm.filemon) : -1;
+#else
+    metafd = -1;
+#endif
     maxfd = -1;
     if (outfd > maxfd)
 	    maxfd = outfd;
